media/css/firefox/sync.less

```diff
@@ -133,6 +137,18 @@
     }
 }
 
+.primary-buttons {
+    display: flex;
+    justify-content: space-around;
```

Comment:
If you have time to review #3765 that would be great :smile: 

If this is a priority I'm happy to rebase my branch however if this merges first